src/app/person/gender.cpp

```diff
@@ -2,12 +2,32 @@
 
 std::string toString(const Gender gender)
 {
+    std::string gender_text{};
     switch (gender)
     {
     case Gender::Male:
-        return "male";
+        gender_text = "male";
+        break;
 
     case Gender::Female:
-        return "female";
+        gender_text = "female";
+        break;
+    default:
+        break;
```

Comment:
if we use it like that maybe its good to stay consistent with explicit assignment and unknown would be assigned

Reply:
done ( mistake )